source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
+on in the event of a tie. 
+
+The following code snippet shows how to specify a descending
+sort on the ``letter`` field followed by the ``_id`` field: 
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("letter", "_id"));
+
+
+The above code snippet returns the documents in the
+:ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`  
+collection in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+Combining Sort Criteria
+-----------------------
+
+To combine sort criteria, use the ``Sorts.orderBy()``
+static factory method. The ``orderBy()`` method applies additional
+sort criteria from left to right in the event of ties. 
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
+descending sort on ``letter`` with an ascending sort on ``_id``.
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.orderBy;
+
+   // <MongoCollection setup code here>
+
+   Bson orderBySort = orderBy(descending("letter"), ascending("_id"));
+   collection.find().sort(orderBySort);
+
+The above code snippet returns the documents in the
+:ref:`sort_example <docs_java_fundamentals_builders_sort_sort_example>`
+collection in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+
+Text Score
+----------
+
+If you are using :manual:`$text </reference/operator/query/text/>`
+queries to search your data you can sort the results of those
+queries by text score (how well your search matched the returned document).
+To sort your results by the
+:manual:`$meta </reference/operator/aggregation/meta/#exp._S_meta>`
+``textScore`` associated with a ``$text`` query, use the
+``Sorts.metaTextScore()`` static factory method to retrieve a sort specification
+for a descending sort by score (best match first).
+
+.. note::
+
+  If your MongoDB instance is running MongoDB 4.2 and earlier,
+  to use a ``Sorts.metaTextScore()``
+  expression with ``FindIterable.sort()``, you must also
+  include the same expression in a projection. This is demonstrated
+  in the code example below.
+
+The ``$text`` command performs a text search on the content indexed by the 
+:manual:`text index </core/index-text/>` of your collection. You must create
```

Comment:
1) This sentence is helpful, but I think could be much more effective in the first paragraph.  
2) Is $text a command or query operator?

Reply:
Thanks for the note. Moved the sentence into the above paragraph (seems to be logically related to ideas presented there), and corrected language around `$text` query operator. Fix: https://github.com/mongodb/docs-java/pull/59/commits/65765202ba4cf66fb44e5aa8d7b5cbb1c21f25cf